src/components/Filter/FilterInput.vue

```diff
@@ -93,11 +93,13 @@
             aria-label="Reset Filter"
             class="filter__delete-button"
             @click="resetFilters(true)"
+            @keydown.enter.exact.stop="resetFilters(true)"
```

Comment:
`@click` does get fired on `keydown.enter`, but adding this will explicitly prevent collisions with `enter` events from parent components. 

I had the problem that `@keydown.enter.exact="handleKeyEnter"` on `QuickNavigationModal` was getting called instead of `resetFilters`. When `FilterInput` was used only by `NavigatorCard` this wasn't a problem, but now is being reused, and having this line allows to keep the intended behavior no matter what the parent component `enter` event modifier is, making the component more generic IMO.